recipes/minio-operator.yaml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/minio/operator/tree/master/helm/minio-operator
+apiVersion: v1
+kind: kbrew
+app:
+  pre_install:
+  - steps:
+    - |
+      # Prerequisites
+      # https://github.com/minio/operator#prerequisites
+      minK8sVersion="v1.19.0"
+      expected=$(echo $minK8sVersion |  sed 's/v//g' | sed 's/\.//g')
+      k8sVersion=$(kubectl version --short=true --output json | jq -r ".serverVersion.gitVersion" | sed 's/-.*//g' | sed 's/v//g' | sed 's/\.//g')
```

Comment:
Can this be done without `jq` ?

Reply:
We are assuming folks will have `jq` as a dependency?

Reply:
I couldn't come up with a better way to find the server version. Due to time crunch, I went ahead with having `jq` dep. But I am sure we can find a better way